exercises/concept/need-for-speed/.docs/introduction.md

```diff
@@ -1,18 +1,46 @@
 # Introduction
 
-In Go, a `struct` is a sequence of named elements called _fields_, each field having a name and type. The name of a field must be unique within the struct. `Structs` can be compared with the _class_ in the Object Oriented Programming paradigm.
+In Go, a `struct` is a sequence of named elements called _fields_, each field has a name and type. The name of a field must be unique within the struct.
+`Structs` can be compared with _classes_ in the Object-Oriented Programming paradigm.
 
-You create a new struct by using the `struct` keyword, a **_built-in type_**, and explicitly define the name and type of the fields as shown in the example below.
+You create a new struct by using the **_built-in type_** keywords `type` and `struct`, and explicitly define the name and type of the fields.
+For example, to define a `Person` struct:
 
 ```go
-type StructName struct{
-    field1 fieldType1
-    field2 fieldType2
+type Person struct{
+    name string
+    age int
 }
 ```
 
-Struct fields are accessed using a `.` notation.
+Once you have defined the `struct`, you need to create a new instance defining the fields _in order_, for example:
 
 ```go
-someStruct.someField = "someValue"
+person := Person{
+	"John",
+	20,
+}
+```
+
+Or you can explicitly define the field name, in any order:
+
+```go
+anotherPerson := Person{
+	age: 22,
+	name: "Anna",
+}
+```
+
+To access instance fields, use the `.` notation. For example, to print the name and age of a `Person`
+you can do the following:
+
+```go
+fmt.Printf("name: %s age: %d\n", person.name, person.age)
+// Output: name: John age: 22
+```
+
+Instance fields can be modified by accessing each field separately:
+
+```go
+anotherPerson.name = "Mary"
```

Comment:
Maybe shorten this slightly to:

````suggestion
To read or modify instance fields, use the `.` notation:

```go
// Update the age
person.age = 23

fmt.Printf("name: %s age: %d\n", person.name, person.age)
// Output: name: John age: 23
```

Instance fields can be modified by accessing each field separately:
````